src/backend/joanie/core/utils/contract_definition.py

```diff
@@ -0,0 +1,56 @@
+"""Utility to `generate document context` data"""
+from django.utils.translation import gettext as _
+
+from joanie.core.utils import image_to_base64
+
+
+# pylint: disable=import-outside-toplevel
+def generate_document_context(contract_definition, user, order=None):
+    """
+    Return a dictionary with placeholder values for the template of the contract definition.
+    It can be used to preview the contract definition, or to generate its context.
```

Comment:
The docstring should be updated as this method does not return only context with placeholder values now.